src/input.rs

```diff
@@ -174,6 +178,17 @@ impl Action {
                         warn!("Error loading data from clipboard. {}", Red(err));
                     });
             },
+            Action::Command(ref program, ref args) => {
+                trace!("running command: {} {}", program, args.join(" "));
```

Comment:
Rather than use `args.join(" ")` here, maybe we can just pass `args` instead and use the debug formatter `{:?}` instead of `{}`.